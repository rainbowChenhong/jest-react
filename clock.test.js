// Copyright 2004-present Facebook. All Rights Reserved.
/* eslint-disable no-unused-vars */

'use strict';

import React from 'react';
import Clock from './Clock';
import renderer from 'react-test-renderer';

jest.useFakeTimers();
Date.now = jest.fn(() => 1482363367071);

it('Clock is renders correctly', () => {
  const tree = renderer.create(<Clock />).toJSON();
  expect(tree).toMatchSnapshot();
});
