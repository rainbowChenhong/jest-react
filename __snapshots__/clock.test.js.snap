// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Clock is renders correctly 1`] = `
<p>
  1482363367.071
   seconds have ellapsed since the UNIX epoch1.
</p>
`;

exports[`renders correctly 1`] = `
<p>
  1482363367.071
   seconds have ellapsed since the UNIX epoch1.
</p>
`;
